### PR TITLE
Add documentation about the optional component functionality

### PR DESCRIPTION
**Pull request by mszostok**
**Description**

Changes proposed in this pull request:

- Add documentation about the optional component functionality

**Related issue(s)**

- https://github.com/kyma-incubator/compass/issues/740

### CODE REVIEW

File: components/kyma-environment-broker/README.md [hunk 1]
```diff
@@ -32,3 +32,49 @@ This table lists the environment variables, their descriptions, and default valu
 | **APP_DATABASE_PORT** | Defines the database port. | `5432` |
 | **APP_DATABASE_NAME** | Defines the database name. | `broker` |
 | **APP_DATABASE_SSL** | Specifies the SSL Mode for PostgrSQL. See all the possible values [here](https://www.postgresql.org/docs/9.1/libpq-ssl.html).  | `disable`|
+
+
+## Development
+
+This section presents how to add or remove functionality for disabling components.
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/README.md]: I would use "runtime component". Somebody could think that a component is a part of KEB.

File: docs/kyma-environment-broker/03-01-runtime-components.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Runtime components
+
+The diagram and steps describe the Kyma Environment Broker (KEB) workflow for calculating and processing runtime components:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-runtime-components.md]: If we decide to merge these two docs into one, we will need to rephrase the introduction.

File: docs/kyma-environment-broker/03-01-runtime-components.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+![runtime-components-architecture](assets/runtime-components.svg)
+
+1. During initialization, KEB reads two files that contain lists of components to be installed in a Runtime:
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-runtime-components.md]: Initialization?

Comment (mszostok) [reply docs/kyma-environment-broker/03-01-runtime-components.md]: during application (KEB) start up.

**Pull request merged**